packages/quantic/cypress/integration/summary/summary.cypress.ts

```diff
@@ -0,0 +1,123 @@
+import {configure} from '../../page-objects/configurator';
+import {
+  InterceptAliases,
+  interceptSearch,
+  interceptSearchIndefinitely,
+} from '../../page-objects/search';
+import {SummaryExpectations as Expect} from './summary-expectations';
+import {getNextResults} from '../../page-objects/actions/action-get-next-results';
+import {
+  setPageSizeValue,
+  setResultsPerPage,
+} from '../../page-objects/actions/action-set-results-per-page';
+
+describe('quantic-summary', () => {
+  const summaryUrl = 's/quantic-summary';
+  const defaultNumberOfResults = 10;
+
+  function visitSummary(waitForSearch = true) {
+    interceptSearch();
+    cy.visit(summaryUrl);
+    configure();
+    if (waitForSearch) {
+      cy.wait(InterceptAliases.Search);
+    }
+  }
+
+  function setupWithPauseBeforeSearch() {
+    interceptSearchIndefinitely();
+    cy.visit(summaryUrl);
+    configure();
+  }
+
+  function loadFromUrlHash(urlHash: string) {
+    interceptSearch();
+
+    cy.visit(`${summaryUrl}#${urlHash}`);
+    configure();
+  }
+
+  it('should not render before results have returned', () => {
+    setupWithPauseBeforeSearch();
+
+    Expect.displaySummary(false);
+  });
+
+  describe('when loading default summary', () => {
+    it('should work as expected', () => {
+      visitSummary(false);
+
+      cy.wait(InterceptAliases.Search).then((interception) => {
+        Expect.displaySummary(true);
+        Expect.displayQuery(false);
+        Expect.displayRange(true);
+        Expect.rangeContains(`1-${defaultNumberOfResults}`);
+        Expect.displayTotal(true);
+        Expect.totalContains(interception.response?.body.totalCount);
+      });
+    });
+  });
+
+  describe('when a query yields no results', () => {
+    it('should work as expected', () => {
+      const url =
+        'q=nowaythisquerywillevereverevertreturnanythingitsimpossible';
```

Comment:
```suggestion
        'q=nowaythisquerywilleverevereverreturnanythingitsimpossible';
```
There is a typo :stuck_out_tongue_winking_eye: 

> itsimpossible

Coveo Cloud could eventually return results for this query even if this keyword is not present in any document. The most reliable way to validate the "no results" case would be to intercept the search request and modify the response to actually return no results.